ui/admin/app/styles/app.scss

```diff
@@ -452,3 +454,103 @@
     }
   }
 }
+
+// secret editor
+.json-secret-editor {
+  .CodeMirror {
+    height: 210px;
+  }
+
+  .CodeMirror-gutters {
+    background: #31332b;
+  }
+}
+
+.secret-editor {
+  background: #272822;
+  border-radius: 4px;
+  height: 210px;
+  margin-bottom: sizing.rems(l);
+}
+
+.skeleton-wrapper {
+  color: var(--ui-gray);
+  display: flex;
+  flex-direction: row;
+  position: relative;
+}
+
+.skeleton-gutter {
+  background: #31332b;
+  border-radius: 4px 0 0 4px;
+  height: 210px;
+  width: 52px;
+}
+
+.skeleton-body {
+  width: 100%;
+
+  > div {
+    background: #31332b;
+    border-radius: 16px;
+    height: 8px;
+    margin: 10.8px;
+
+    &:first-child {
+      margin-top: 16px;
+    }
+
+    &.s {
+      width: 5%;
+    }
+
+    &.m {
+      width: 10%;
+    }
+
+    &.l {
+      width: 15%;
+    }
+
+    &.xl {
+      width: 20%;
+    }
+  }
+}
+
+.details-view {
+  font-size: 14px;
+  left: 50%;
+  margin: -12px 0 0 -165.5px;
+  position: absolute;
+  top: 50%;
+  z-index: 1;
+
+  .rose-icon {
+    #{--size}: sizing.rems(m);
```

Comment:
Is this class and work around even needed, how come we couldn't have just set passed a `@size='medium'` to the rose icon?

Also are we continuing to use rose icon or are we swapping to the new hds flight icons?

Reply:
😮‍💨 I got lost in the weeds of editing styles and just did it here, forgetting I can just set `@size`. Fixed now.